appclient/src/main/java/org/jboss/as/appclient/subsystem/parsing/AppClientXml.java

```diff
@@ -119,6 +119,9 @@ public void readElement(final XMLExtendedStreamReader reader, final List<ModelNo
             case DOMAIN_17_0:
                 readServerElement_1_1(readerNS, reader, address, operationList);
                 break;
+            case DOMAIN_18_0:
```

Comment:
We don't need a specific case for 18 adding, we just need to call the new method in the for loop below which catches any version not listed above - i.e. all versions after 17.

Reply:
Fixed - I'll push after CI kill or completion